src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -1,3 +1,3 @@
 databaseChangeLog:
-  - includeAll:
-      path: db/changelog/changes
+  - include:
+      file: db/changelog/changes/02-create-roles-table.yaml
```

Comment:
The past code will be validly work so that we do not write constantly our liquibase scripts.